changelog.json

```
@@ -1,5 +1,54 @@
 {
   "releases": {
+    "1.7.1": [
+      "[New] You can now choose to bring your changes with you to a new branch or stash them on the current branch when switching branches - #6107",
+      "[New] Rebase your current branch onto another branch using a guided flow - #5953",
+      "[New] Recently opened repositories appear at the top of the repository list - #7132",
+      "[New] Suggested next steps now includes suggestion to create a pull request after publishing a branch - #7505",
+      "[Added] .resx syntax highlighting - #7235. Thanks @say25!",
+      "[Added] \"Exit\" menu item now has accelerator and access key - #6507. Thanks @AndreiMaga!",
+      "[Added] Help menu entry to view documentation about keyboard shortcuts  - #7184",
+      "[Added] \"Discard all changes\" action under Branch menu - #7394. Thanks @ahuth!",
+      "[Fixed] \"Esc\" key does not close Repository or Branch list - #7177. Thanks @roottool!",
+      "[Fixed] Warn when renaming a branch with a stash - #7283",
+      "[Fixed] Attempting to revert commits not on current branch results in an error - #6300. Thanks @msftrncs!",
+      "[Fixed] Clicking \"Undo\" doesn't repopulate summary in commit form - #6390. Thanks @humphd!",
+      "[Fixed] Crash when unable to create log file on disk - #7096",
+      "[Fixed] Emoji rendering in app when account name has special characters - #6909",
+      "[Fixed] Error when selecting diff text while diff is updating - #7131",
+      "[Fixed] Files staged outside Desktop for deletion are incorrectly marked as modified after committing - #4133",
+      "[Fixed] Horizontal scroll bar appears unnecessarily when switching branches - #7212",
+      "[Fixed] Icon accessibility labels fail when multiple icons are visible at the same time - #7174",
+      "[Fixed] Incorrectly encoding URLs affects issue filtering - #7506",
+      "[Fixed] License templates do not end with newline character - #6999",
+      "[Fixed] Merge and rebase conflicts banners do not hide after aborting operation outside Desktop - #7046",
```

Could _Merge_ and _Rebase conflicts_ be quoted and sentence-cased to better clarify that they are banners?

---

Could _Merge_ and _Rebase conflicts_ be quoted and sentence-cased to better clarify that they are banners?

---

I think this could just be simplified down this:

```suggestion
      "[Fixed] Conflicts banners do not hide after aborting operation outside Desktop - #7046",
```

I've been using quote marks to indicate proper names for things, and I didn't think it fitted nicely with the phrasing. Moving to the suggested change would bypass the issue completely. Thoughts?

---

Yea, I think _Conflicts banners_ is pretty clear.